package.json

```diff
@@ -36,19 +35,19 @@
     "test:e2e:playground": "cypress run --spec cypress/integration/playground/**/*.js",
     "test:e2e:template-starter": "cypress run --spec cypress/integration/template-starter/**/*.js",
     "vrt:components": "percy exec -- yarn test:visual",
-    "playground:build": "yarn --cwd playground build",
-    "playground:start": "yarn --cwd playground start",
-    "playground:start:prod:local": "yarn --cwd playground start:prod:local",
-    "visual-testing-app:start": "yarn --cwd visual-testing-app start",
-    "visual-testing-app:build": "yarn --cwd visual-testing-app build",
-    "visual-testing-app:serve": "yarn --cwd visual-testing-app serve",
-    "template-starter:build": "yarn --cwd application-templates/starter build",
-    "template-starter:start": "yarn --cwd application-templates/starter start",
-    "template-starter:start:prod:local": "yarn --cwd application-templates/starter start:prod:local",
-    "prebuild": "lerna run prebuild --no-private",
-    "build": "preconstruct build && NODE_ENV=production lerna run build --no-private",
-    "build:watch": "preconstruct watch & NODE_ENV=development lerna run --no-private --parallel build:bundles:watch",
-    "build:website": "yarn build && yarn --cwd website build && yarn --cwd website-components-playground build",
+    "playground:build": "manypkg run playground build",
+    "playground:start": "manypkg run playground start",
+    "playground:start:prod:local": "manypkg run playground start:prod:local",
+    "visual-testing-app:start": "manypkg run visual-testing-app start",
+    "visual-testing-app:build": "manypkg run visual-testing-app build",
+    "visual-testing-app:serve": "manypkg run visual-testing-app serve",
+    "template-starter:build": "manypkg run application-template-starter build",
+    "template-starter:start": "manypkg run application-template-starter start",
+    "template-starter:start:prod:local": "manypkg run application-template-starter start:prod:local",
+    "prebuild": "yarn compile-intl",
+    "build": "preconstruct build && NODE_ENV=production NODE_ENV=production manypkg run mc-html-template build && NODE_ENV=production manypkg run mc-scripts build",
```

Comment:
Ok so we only have those 2 packages that are compiled through babel.

Let's keep them hardcoded here for now and follow up on this. We can look into contributing to manypkg to have a more flexible `run` command, or have a custom script that automatically detects which packages should be built using babel.

Reply:
I opened and issue with them to propose it. I somehow find this setup loads easier than our prior mix of tools which did stuff.